src/main/MQ2Utilities.cpp

```diff
@@ -73,12 +49,11 @@ static void DebugSpewImpl(bool always, bool logToFile, const char* szFormat, va_
 
 	vsprintf_s(szOutput, theLen, szFormat, vaList);
 
-	strcat_s(szOutput, theLen, "\n");
```

Comment:
this newline is required for proper output through OutputDebugString.

Reply:
it results in extra blank lines in the DebugLog however.  Where does the output from OutputDebugString() go? is that purely for VS debugging?

Reply:
it goes into the process debug stream, which is viewable through tools like DbgView or an attached debugger. I personally use it extensively, so it cannot be made into garbage.